src/app/feature/campaign/pages/campaign-debug/components/campaign-variables-editor.ts

```diff
@@ -34,7 +34,7 @@ import { TemplateFieldService } from "src/app/shared/components/template/service
         </ion-label>
         <ion-datetime
           #firstLaunch
-          (ionChange)="setAppLaunchData(firstLaunch.value, appDay.value)"
+          (ionChange)="setAppLaunchData($any(firstLaunch.value), appDay.value)"
```

Comment:
minor type issue because datetime can output string or string[] depending on configuration. So just cast to `any` type (as we know the output is string format)